blivet/udev.py

```diff
@@ -39,7 +39,7 @@
 global_udev = pyudev.Context()
 log = logging.getLogger("blivet")
 
-device_name_blacklist = []
+ignored_device_names = []
```

Comment:
Hi, Anaconda is using this variable, so we should coordinate the releases. See: https://github.com/rhinstaller/anaconda/pull/2698

Reply:
Oh, I didn't notice this one. We should probably also only deprecate this one and remove it in 3.3.

Reply:
I agree, but I don't think our decorator will work on variables. Any ideas on how to indicate that it is deprecated? The only idea I have is to add the to-be-removed attributes to the API docs (they aren't there now AFAIK) with a note about their deprecation and impending removal.

Reply:
Maybe we should add a note to docs for deprecated items. That will both make the deprecations more visible and provide a way to indicate deprecation of a variable.

Reply:
> but I don't think our decorator will work on variables

We could do something really crazy like create a subclass of `list` and decorate its `insert` method, but that I think adding a comment a mentioning the deprecation in documentation should be good enough.

Reply:
It turns out none of the names I changed in this pull request appear in the API docs. I'm not sure if I want to add this to the API just so we can deprecate/remove it. Maybe we should fall back on the original suggestion to simply coordinate the releases with anaconda.

Reply:
In that case I guess we can just do this without the deprecation. I've already created 3.3-devel branch so let's just move this to 3.3.